### PR TITLE
Add saml_id to the account import actions

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/actions/user/json_upload.py [hunk 1]
```diff
@@ -122,6 +130,22 @@ def generate_entry(
...
+                    if searchdata := self.get_search_data(payload_index):
+                        entry["saml_id"] = {
+                            "value": entry["saml_id"],
+                            "info": "done",
+                            "id": searchdata["id"],
+                        }
+                    else:
+                        entry["saml_id"] = {"value": entry["saml_id"], "info": "done"}
+                        state = ImportState.ERROR
+                        messages.append(f"Duplicate in csv list index: {payload_index}")
```
Comment (jsangmeister) [inline openslides_backend/action/actions/user/json_upload.py:145]: Why is this a duplicate? Couldn't it be that the user just didn't exist yet?

Also you can unify this whole `saml_id` section with the `username` one above it.

File: openslides_backend/action/actions/user/json_upload.py [hunk 2]
```diff
@@ -153,6 +177,10 @@ def generate_entry(
                         "info": ImportState.GENERATED,
                     }
             self.handle_default_password(entry, state)
+            if entry.get("saml_id", {}).get("value"):
+                entry.pop("password", None)
+                entry.pop("default_password", None)
```
Comment (jsangmeister) [inline openslides_backend/action/actions/user/json_upload.py:182]: A warning would be good that the fields were removed

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/user/json_upload.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/user/json_upload.py]: Unified.

File: openslides_backend/action/actions/user/json_upload.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/user/json_upload.py:168]: Add a message to the messages, if one field is removed.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**